dbt/models/qc/schema.yml

```diff
@@ -486,3 +486,124 @@ models:
           - dbt_utils.accepted_range:
               name: qc_vw_neg_asmt_value_valasm3_gte_0
               <<: *test-non-negative
+              
+  - name: qc.vw_iasworld_sales_null_values
+    description: '{{ doc("view_vw_iasworld_sales_null_values") }}'
+    config:
+      tags:
+        - test_qc_sales
+    tests:
+      - expression_is_true:
+          name: qc_vw_iasworld_sales_null_values_deed_null_count_lte_1000
+          expression: deed_null_count < 1000
+      - expression_is_true:
+          name: qc_vw_iasworld_sales_null_values_buyer_null_count_lte_1000
+          expression: buyer_null_count < 1000
+      - expression_is_true:
+          name: qc_vw_iasworld_sales_null_values_seller_null_count_lte_1000
+          expression: seller_null_count < 1000
+      - expression_is_true:
+          name: qc_vw_iasworld_sales_null_values_price_null_count_lte_1000
+          expression: price_null_count < 1000
+
+  - name: qc.vw_sale_mydec_null_values
+    description: '{{ doc("view_vw_sale_mydec_null_values") }}'
+    config:
+      tags:
+        - test_qc_sales
+    tests:
+      - expression_is_true:
+          name: qc_vw_sale_mydec_null_values_deed_null_count_lte_1000
+          expression: deed_null_count < 1000
+      - expression_is_true:
+          name: qc_vw_sale_mydec_null_values_buyer_null_count_lte_1000
+          expression: buyer_null_count < 1000
+      - expression_is_true:
+          name: qc_vw_sale_mydec_null_values_seller_null_count_lte_1000
+          expression: seller_null_count < 1000
+      - expression_is_true:
+          name: qc_vw_sale_mydec_null_values_price_null_count_lte_1000
+          expression: price_null_count < 1000
+      - expression_is_true:
+          name: qc_vw_sale_mydec_null_values_address_null_count_lte_1000
+          expression: address_null_count < 1000
+
+  - name: qc.vw_iasworld_sale_rowcount_matches_sales_mydec
+    description: '{{ doc("view_vw_iasworld_sale_rowcount_matches_sales_mydec") }}'
+    config:
+      tags:
+        - test_qc_sales
+    tests:
+      - expression_is_true:
+          name: qc_vw_iasworld_sale_rowcount_matches_sales_mydec_check
+          expression: IasWorld_sales > 0.95 * my_dec_sales and IasWorld_sales < 1.05 * my_dec_sales
```

Comment:
**[Thought, non-blocking]** Just thinking out loud here -- I think there are two approaches we could take to implementing these checks. Either:

1. We implement the conditional logic that checks for incorrect rows in the `expression` field, as is done here; or
2. We implement the conditional logic in the body of the query, and then query for that value in the test (in this case, that would mean checking for `comparison = 'No significant difference'`)

It seems to me that the tradeoff between these two approaches is that the code for 1) is a bit easier to read, since the test as defined in `schema.yml` is very clear about how its logic works; but on the other hand, the _results_ for 2) will be easier to read in the case of failing tests, since we can store failing rows with explicit error messages like `IasWorld 5% Higher`.

I think the way it's written is fine to merge for now, but I wonder whether approach 2) will start to become more appealing as I audit test failure output in service of https://github.com/ccao-data/data-architecture/issues/263. Either way, nothing to change here, I just wanted to preserve my thinking on this for posterity!